src/azurelinux/manifest.json

```
@@ -943,8 +941,21 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/azurelinux/3.0/build-deps",
+              "os": "linux",
+              "osVersion": "azurelinux3.0",
+              "tags": {
+                "azurelinux-3.0-build-deps-amd64": {}
```

I do think the tag and dockerfile path should reflect the .NET version. There may be changes in runtime-deps that would break the usage of an older .NET version.

```suggestion
              "dockerfile": "src/azurelinux/3.0/net10.0/build-deps",
              "os": "linux",
              "osVersion": "azurelinux3.0",
              "tags": {
                "azurelinux-3.0-net10.0-build-deps-amd64": {}
```